rholang/src/main/scala/coop/rchain/rholang/interpreter/Reduce.scala

```diff
@@ -473,8 +501,10 @@ class DebruijnInterpreter[M[_], F[_]](
       }
     }
 
-    charge[M](newBindingsCost(neu.bindCount)) >>
-      alloc(neu.bindCount, neu.uri).flatMap(eval(neu.p)(_, rand, sequenceNumber))
+    spanM.trace(newSpanLabel) {
```

Comment:
Same here and in other places. Allocations of helper functions contribute to the measured operation's duration, so they should be included in the span

Reply:
Guess that's another point for having them as private methods :)